src/test/java/uk/gov/hmcts/reform/roleassignment/auditlog/AuditInterceptorTest.java

```diff
@@ -65,7 +66,7 @@ void shouldPrepareAuditContextWithHttpSemanticsLongResponse() {
 
         assertThat(auditContextSpy.getHttpMethod()).isEqualTo(METHOD);
         assertThat(auditContextSpy.getRequestPath()).isEqualTo(REQUEST_URI);
-        assertThat(auditContextSpy.getHttpStatus()).isEqualTo(STATUS);
+        assertThat(auditContextSpy.getHttpStatus()).isEqualTo(422);
         assertThat(AuditContextHolder.getAuditContext()).isNull();
```

Comment:
look like you changes the status value to 422 [STATUS = 422;] the what is the reason to change here as 422?

Reply:
I reverted this - I thought I had to change the scenario to match the outcome of the new 422 response.